utils/network.go

```diff
@@ -179,7 +176,7 @@ func (n *Network) ProtocolID() protocol.ID {
 }
 
 func knownNetworkNames() []string {
-	networks := []Network{Mainnet, Goerli, Goerli2, Integration, Sepolia, SepoliaIntegration}
```

Comment:
do we need to remove Goerli, Goerli2 vars ?

Reply:
yes, from this slice error message is constructed [here](https://github.com/NethermindEth/juno/blob/2697c3c7fb2f097e09fd5ca169c8855d193eacd9/utils/network.go#L16):

Reply:
I meant `utils.Goerli` and `utils.Goerli2` that contains information about these networks

Reply:
Ah, ok. If I got Aneeque correctly I was supposed to prevent running node connected to these networks (CLI-level change). 
Removing module variables e.g. `utils.Goerli` requires cleaning bunch of tests - a bit more work.